JavaIntegration/src/main/java/org/openzen/zencode/java/module/JavaNativeTypeTemplate.java

```diff
@@ -194,18 +197,42 @@ private void loadMethods() {
 				continue;
 
 			FunctionHeader header = headerConverter.getHeader(typeVariableContext, method);
-
-			if (expansion && !isStaticExpansion) {
+			if (expansion && !isStaticExpansionMethod) {
 				FunctionParameter[] withoutFirst = Arrays.copyOfRange(header.parameters, 1, header.parameters.length);
 				header = new FunctionHeader(header.typeParameters, header.getReturnType(), header.thrownType, withoutFirst);
 			}
 
 			for (MethodID id : ids) {
-				JavaRuntimeMethod runtimeMethod = new JavaRuntimeMethod(class_, target, method, id, header, implicit, expansion && !isStaticExpansion);
+				JavaRuntimeMethod runtimeMethod = new JavaRuntimeMethod(class_, target, method, id, header, implicit, expansion && !isStaticExpansionMethod);
 				methods.computeIfAbsent(id, x -> new ArrayList<>()).add(runtimeMethod);
 				class_.module.getCompiled().setMethodInfo(runtimeMethod, runtimeMethod);
 			}
 		}
+
+
+		if (class_.cls.isEnum()) {
+			Stream.of(
+					BuiltinMethodSymbol.ENUM_NAME,
+					BuiltinMethodSymbol.ENUM_ORDINAL,
+					//BuiltinMethodSymbol.ENUM_VALUES,
+					BuiltinMethodSymbol.ENUM_COMPARE
+			).forEach(method -> methods
+					.computeIfAbsent(method.getID(), x -> new ArrayList<>())
+					.add(method)
+			);
+
+			try {
+				MethodID id = MethodID.staticGetter("values");
+				FunctionHeader header = new FunctionHeader(new ArrayTypeID(target));
+				Method method = class_.cls.getMethod("values");
```

Comment:
❓ Question: Should this be placed in the for-loop above as special `if` check alongside the `hasAnnotation(Getter.class)` or should it be placed down here an be near the other enum-builtin registrations?